src/Azure.Functions.Cli/Actions/UserInputHandler.cs

```diff
@@ -160,6 +160,10 @@ private string LabelMap(string label)
                 { "$blobTrigger_path_label", "Container Path" },
                 { "$blobTrigger_connection_label", "Storage Account Connection String" },
                 { "$eventGrid_name_label", "EventGrid Name" },
+                { "$chat_model_name_label", "Chat Model Name" },
```

Comment:
Do we need to add/update any tests for this change?

Reply:
Ideally these labels should not be hardcoded. For now this is a temporary fix. We should be reading these labels from the templates. 
Created an issue for it https://github.com/Azure/azure-functions-core-tools/issues/3746